src/types/File.coffee

```diff
@@ -9,6 +9,12 @@
 		if Buffer.isBuffer(value)
 			callback(null, value)
 		else if _.isString(value)
+			if value.length % 2 != 0
+				callback('could not be converted to binary: hex string must have an even length')
+				return
```

Comment:
It's personal taste but, if you are using `return` you can omit the `else` afterwards, I personally always omit it because it's redundant